relay-server/src/utils/multipart.rs

```diff
@@ -276,25 +299,41 @@ impl futures::Stream for LimitedField<'_> {
         }
     }
 }
+#[allow(dead_code)]
+pub struct UnconstrainedMultipart(pub Multipart<'static>);
```

Comment:
This struct could use some documentation to explain what makes it different from a `Remote<Multipart>`. Also, personally I still think it would be neat if there were a `ConstrainedMultipart` that takes the place of the current `Remote<Multipart>`, but it's not necessary.